src/View/Helper/FormHelper.php

```diff
@@ -31,7 +31,6 @@ class FormHelper extends Helper
         'help' => '<div class="help-block">{{content}}</div>',
         'inputContainer' => '<div class="form-group {{type}}{{required}}">{{content}}{{help}}</div>',
         'inputContainerError' => '<div class="form-group {{type}}{{required}} has-error">{{content}}{{error}}{{help}}</div>',
-        'checkboxWrapper' => '<div class="checkbox"><label>{{input}}{{label}}</label></div>',
         'multipleCheckboxWrapper' => '<div class="checkbox">{{label}}</div>',
```

Comment:
I think `multipleCheckboxWrapper` can also be removed. It doesn't seem to be used by any widget in this plugin or core widgets. Only other reference I can find is [here](https://github.com/FriendsOfCake/bootstrap-ui/blob/master/src/View/Helper/FormHelper.php#L224) where it's overwritten by string contained in `checkboxWrapper`. So that particular line can also be removed.